port/action/schema.go

```diff
@@ -269,10 +269,9 @@ func ActionSchema() map[string]schema.Attribute {
 					MarkdownDescription: "Required when selecting type WEBHOOK. The URL to invoke the action",
 					Required:            true,
 				},
-				"agent": schema.StringAttribute{
+				"agent": schema.BoolAttribute{
```

Comment:
this breaks API, lets revert this change, and please elaborate on why this is needed?

Reply:
okay. In the description, the client stated that the agent property is supposed to be a boolean instead of a string attribute. we were described the values of the property as bool and yet it was taken a string property.

[related link](https://registry.terraform.io/providers/port-labs/port-labs/latest/docs/resources/port_action#nestedatt--self_service_trigger--user_properties--array_props--boolean_items)

Reply:
this can be also a jq, so this is the reason why its a string and not a boolean, worth elaborating more in the markdown

Reply:
Makes sense. Let me change it back to String and add more content to the description

Reply:
@pazhersh what do you think of this 
```
"agent": schema.StringAttribute{
					MarkdownDescription: "Specifies whether to use an agent to invoke the action. This can be a boolean value (`true` or `false`) or a JQ template if dynamic evaluation is needed.",
					Optional:            true,
				}
```